libs/ledger-live-common/src/families/cosmos/js-prepareTransaction.ts

```diff
@@ -55,24 +55,28 @@ export const getEstimatedFees = async (
 
   const cosmosAPI = new CosmosAPI(account.currency.id);
   const { protoMsgs } = txToMessages(account, transaction);
-  const { sequence, pubKeyType, pubKey } = await cosmosAPI.getAccount(account.freshAddress);
-  const signature = new Uint8Array(Buffer.from(account.seedIdentifier, "hex"));
+  const { sequence, pubKey, pubKeyType } = await cosmosAPI.getAccount(account.freshAddress);
 
-  const txBytes = buildTransaction({
+  const unsignedTx = buildTransaction({
     protoMsgs,
-    memo: transaction.memo || "",
-    pubKeyType,
-    pubKey,
-    feeAmount: undefined,
     gasLimit: undefined,
-    sequence: sequence ? sequence + "" : "0",
-    signature,
+    feeAmount: [
+      {
+        denom: account.currency.units[1].code,
+        amount: "1", // Amount should just not be 0 as it would impact the simulation by requiring less gas
+      },
+    ],
+    memo: "",
```

Comment:
Shouldn't we still use `transaction.memo || ""` ? :thinking: if size matters

Reply:
Yup probably a good idea, I guess with a huge memo it would also change the estimation 🤷‍♂️ 
Will update 👌